common/artifact_spawns/wc_artifact_spawns.txt

```diff
@@ -485,4 +485,253 @@ aluneth = {
     artifacts = {
 		aluneth = { value = 1 }
     }
+}
+felomelorn = {
+	max_amount = 1
+    spawn_date = -1.1.1
+	
+	spawn_chance = {
+		value = 100
+		modifier = {
+			factor = 0
+			NOT = { has_dlc = "Mystics" }
+		}
+	}
+    
+    weight = {
+		value = 0
+		additive_modifier = {
+			value = 100
+			character = 42002		# Anastarian
+		}
+	}
+    artifacts = {
+		felomelorn = { value = 1 }
+    }
+}
+hammer_of_the_twilight = {
+	max_amount = 1
+    spawn_date = 583.1.1
+	
+	spawn_chance = {
+		value = 100
+		modifier = {
+			factor = 0
+			NOT = { has_dlc = "Mystics" }
+		}
+		modifier = {
+			factor = 0
+			NOT = { year = 585 }
+		}
+	}
+    
+    weight = {
+		value = 0
+		additive_modifier = {
+			value = 100
+			is_playable = yes
+			
+			culture = twilights_hammer
+		}
+	}
+    artifacts = {
+		hammer_of_the_twilight = { value = 1 }
+    }
+}
+hammer_of_the_lightbringer = {
+	max_amount = 1
+    spawn_date = 583.1.1
+	
+	spawn_chance = {
+		value = 100
+		modifier = {
+			factor = 0
+			NOT = { has_dlc = "Mystics" }
+		}
+	}
+    
+    weight = {
+		value = 0
+		additive_modifier = {
+			value = 100
+			character = 60036						# Uther Lightbringer
+		}
+	}
+    artifacts = {
+		hammer_of_the_lightbringer = { value = 1 }
+    }
+}
+flute_of_accuracy = {
+	max_amount = 5
+    spawn_date = -1.1.1
+	
+	spawn_chance = {
+		value = 100
+		modifier = {
+			factor = 0
+			NOT = { has_dlc = "Mystics" }
+		}
+	}
+    
+    weight = {
+		value = 0
+		additive_modifier = {
+			value = 100
+			OR = {
+				character = 42086						# Alleria Windrunner
+				character = 42002						# Anasterian Sunstrider
+				character = 42089						# Idorial Windrunner
+			}
+		}
+	}
+    artifacts = {
+		flute_of_accuracy = { value = 1 }
+    }
+}
```

Comment:
Probably it's better just give it to Alleria, since if you will play as Anasterian, there is no point in NOT restarting your game to get 2 artifacts from the start.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/67306007-f0c7b080-f4fe-11e9-9d0b-6469e690d22d.png)

</details>